format_spec/consolidated_commits_file.md

```diff
@@ -0,0 +1,24 @@
+# Consolidated Commits File
+
+A consolidated commits file has name `<timestamped_name>.meta` and is located here:
+
+```
+my_array                              # array folder
+   |_ ....
+   |_ __commits                       # array commits folder
+         |_ <timestamped_name>.meta   # consolidated commits file
```

Comment:
Lets not use `.meta` for this. `.meta` is for consolidated fragment metadata files, so reusing the extension here could lead to confusion for what is inside the file.